docs/provisioning.md

```diff
@@ -0,0 +1,111 @@
+# Provisioning the IDR
+
+## Overview of the architecture
+
+The IDR contains two main groups of servers:
+
+The production (public-facing) IDR (3 servers):
+- Database
+- OMERO.server
+- Nginx gateway
+
+The virtual analysis environment (VAE) IDR (3 servers):
+- Database
+- OMERO.server
+- Docker Manager
+
+
+## Ansible prerequisites
+
+Almost all of the provisioning and deployment in the IDR is done using Ansible 2.1.
+All ansible commands should be run from a shell in the [`ansible`](../ansible) directory.
+
+You must first install the required galaxy roles:
+
+    ansible-galaxy install -r requirements.yml
+
+The [`ansible.cfg`](../ansible/ansible.cfg) configuration file will install the roles into the [`vendor`](../ansible/vendor) directory.
+
+
+## OpenStack
+
+The IDR is currently hosted on OpenStack, see below for an example Ansible playbook for provisioning compute, storage and networking.
+The Ansible openstack modules require the `shade` python module.
+
+This playbook will create two networks `idr` and `idr-a` for the production and analysis servers, and multiple instances and storage volumes.
+
+
+### Production IDR
+Network: `idr`
+
+Instances:
+- `idr-database`: PostgreSQL database server
+- `idr-omero`: OMERO.server, including OMERO.web
+- `idr-proxy`: Nginx gateway with custom caching configuration
+
+Volumes:
+- `idr-database-db`: PostgreSQL data directory
+- `idr-omero-data`: OMERO data directory
+- `idr-proxy-nginxcache`: Nginx cache directory
+
+
+### Analysis IDR
+Network: `idr-a`
+
+Instances:
+- `idr-a-database`: PostgreSQL database server
+- `idr-a-omero`: OMERO.server
+- `idr-a-dockermanager`: A Docker server for running VAEs
+
+Volumes:
+- `idr-a-database-db`: PostgreSQL data directory
+- `idr-a-omero-data`: OMERO data directory
+- `idr-a-dockermanager-jupyter`: Files created by the VAE
+
+Note `idr-proxy` is also connected to the `idr-a` network to provide access to the analysis platform.
```

Comment:
If we're going to put a diagram in I think a proper drawing would be better than a screenshot which isn't particularly clear if you're not familiar with OpenStack. I can do one, do you consider it a blocker?